src/sentry/search/utils.py

```diff
@@ -224,10 +224,11 @@ def parse_user_value(value, user):
         return User(id=0)
 
 
-def get_latest_release(projects, environments):
-    release_qs = Release.objects.filter(
-        organization_id=projects[0].organization_id, projects__in=projects
-    )
+def get_latest_release(projects, environments, organization_id=None):
+    if organization_id is None:
+        organization_id = projects[0].organization_id
```

Comment:
When would you potentially want a different organization_id than what is on the project?

Reply:
In this case its because we only have a list of project_ids but still have access to the organization_id

Reply:
Should we update the other usages of this to also include the organization_id?

Reply:
Other usage of this is in, `GroupEventsEndpoint` which gets the org even for other functions via `group.project.organization`